edb/server/args.py

```diff
@@ -619,6 +624,36 @@ def server_options(func):
     return func
 
 
+_compiler_options = [
+    click.option(
+        "--pool-size",
+        type=int,
+        callback=_validate_compiler_pool_size,
+        default=compute_default_compiler_pool_size(),
+        help=f"Number of compiler worker processes. Defaults to "
+             f"{compute_default_compiler_pool_size()}.",
+    ),
+    click.option(
+        "--client-schema-cache-size",
+        type=int,
+        default=100,
+        help="Number of client schemas each worker could cache at most. The "
+             "compiler server is not affected by this setting, it keeps a "
+             "pickled copy of the client schema of all active clients."
+    ),
+    click.option(
+        "--socket-path",
```

Comment:
This should be `--listen-addresses` / `--listen-port` per the comment re socket above.